core/src/mindustry/world/blocks/storage/CoreBlock.java

```diff
@@ -144,12 +151,11 @@ public void setBars(){
 
     @Override
     public void init(){
-        //assign to update clipSize internally
-        lightRadius = 30f + 20f * size;
-        fogRadius = Math.max(fogRadius, (int)(lightRadius / 8f * 3f) + 13);
-        emitLight = true;
-
         super.init();
+
+        //just to avoid breaking any modded blocks who havn't set a lightRadius/fogRadius post update
+        if(lightRadius == -1) lightRadius = 30f + 20f * size;
```

Comment:
Assigning light radius after init() doesn't update lightClipSize, which breaks the block when it's offscreen.